projects/go-lib/src/lib/components/go-off-canvas/go-off-canvas.component.html

```diff
@@ -6,17 +6,19 @@
 >
 </div>
 <div
-  [@offCanvas]="opened ? 'slideIn' : 'slideOut'"
+  [@offCanvas]="{value: opened ? 'slideIn' : 'slideOut', params: {width: size === 'large' ? '75vw' : '300px'}}"
```

Comment:
What do you think about moving this logic into a method that assigns a local variable in the typescript file? Similar to how we do `[ngClass]` declarations [for buttons](https://github.com/mobi/goponents/blob/c485c620bd94eac87af86dc41b91c1b90d2468f8/projects/go-lib/src/lib/components/go-button/go-button.component.ts#L49-L52).

Reply:
What would be the benefit in doing that? 
These are values that are passed in as inputs and sent directly to the view. Wouldn't dealing with them in the typescript file just add a level of indirection? And then we need to manually manage them in lifecycle hooks, like we are doing there in the button component.

Reply:
I was just thinking it would clean up the html a bit, but you're right. It would probably be more work.